autoload/nvimdev.vim

```diff
@@ -145,9 +147,6 @@ function! nvimdev#init(path) abort
   augroup nvimdev
     autocmd!
     autocmd BufRead,BufNewFile *.h set filetype=c
-    if get(g:, 'nvimdev_auto_lint', 1)
```

Comment:
The README about this:

```
#### `g:nvimdev_auto_lint` (default `0`)

Automatically run `:Neomake` to lint sources after writing buffers.  Disabled
by default since cool people would have this already setup.
```

I must've set it to `1` while I was tinkering with the script.